crates/bevy_render/src/color/mod.rs

```diff
@@ -57,84 +57,84 @@ pub enum Color {
     },
 }
 
-impl Color {
+impl LegacyColor {
     /// <div style="background-color:rgb(94%, 97%, 100%); width: 10px; padding: 10px; border: 1px solid;"></div>
-    pub const ALICE_BLUE: Color = Color::rgb(0.94, 0.97, 1.0);
+    pub const ALICE_BLUE: LegacyColor = LegacyColor::rgb(0.94, 0.97, 1.0);
     /// <div style="background-color:rgb(98%, 92%, 84%); width: 10px; padding: 10px; border: 1px solid;"></div>
```

Comment:
is there a reason to not use Self in the type assign of this consts? could simplify further work on this area maybe.

Reply:
Yeah, it's not a bad choice. I wanted to keep this PR as simple as possible though.